osu.Game/Overlays/Direct/PlayButton.cs

```diff
@@ -31,18 +29,21 @@ public BeatmapSetInfo BeatmapSet
                 if (value == beatmapSet) return;
                 beatmapSet = value;
 
+                if (Preview != null)
+                {
+                    Preview.Stop();
+                    RemoveInternal(Preview);
```

Comment:
will this fail if the `Preview` hasn't been added yet?

Reply:
no